Gems/PhysX/Code/Source/PhysXCharacters/Components/CharacterGameplayComponent.cpp

```diff
@@ -145,8 +133,7 @@ namespace PhysX
 
         if (state.touchedActor != nullptr || (state.collisionFlags & physx::PxControllerCollisionFlag::eCOLLISION_DOWN) != 0)
```

Comment:
I'm wondering if we should get rid of this check as well and go straight to the overlap query.